### PR TITLE
Show dropdown text if controls and volume is hidden (#149)

### PR DESCRIPTION
**Pull request by bernimoses**

### CODE REVIEW

**Comment by kalkih** [comment]
Build fails because the bitwise operator is not allowed in the [airbnb javascript style guide](https://github.com/airbnb/javascript) which is what this project is based on.

**Comment by bernimoses** [comment]
I added a exception for those two lines since the alternative is much less readable. Is that ok?

File: src/components/powerstrip.js [hunk 1]
```diff
@@ -33,11 +33,15 @@ class MiniMediaPlayerPowerstrip extends LitElement {
   }
 
   get sourceSize() {
-    return (this.config.source === 'icon' || this.config.collapse || this.idle);
+    return (this.config.source === 'icon' || (this.config.hide.controls ^ this.config.hide.volume) || this.idle); // eslint-disable-line no-bitwise
```
Comment (kalkih) [inline src/components/powerstrip.js]: You should be able to do `(this.config.hide.controls !== this.config.hide.volume)` instead

File: src/components/powerstrip.js [hunk 2]
```diff
@@ -33,11 +33,15 @@ class MiniMediaPlayerPowerstrip extends LitElement {
...
+    return (this.config.source === 'icon' || this.config.hide.controls !== this.config.hide.volume || this.idle);
   }
 
   get soundSize() {
-    return (this.config.sound_mode === 'icon' || this.config.collapse || this.idle);
+    return (this.config.sound_mode === 'icon' || this.config.hide.controls !== this.config.hide.volume || this.idle);
+  }
+
+  get hasControls() {
+    return (this.config.collapse && this.player.active && !this.config.hide.controls);
```
Comment (kalkih) [inline src/components/powerstrip.js]: <mmp-media-controls> contains both volume & playback controls, so we need to check both `this.config.hide.controls` & `this.config.hide.volume` here.

Comment (bernimoses) [reply src/components/powerstrip.js]: Fixed, had to reformat because of max line length.

File: src/components/powerstrip.js [hunk 3]
```diff
@@ -33,11 +33,22 @@ class MiniMediaPlayerPowerstrip extends LitElement {
...
+    return (this.config.sound_mode === 'icon'
+      || this.config.hide.controls !== this.config.hide.volume
+      || this.idle);
+  }
+
+  get hasControls() {
+    return (this.config.collapse
+      && this.player.active
+      && !this.config.hide.controls
+      && !this.config.hide.volume);
```
Comment (kalkih) [inline src/components/powerstrip.js]: What if only one of them is hidden?

Comment (bernimoses) [reply src/components/powerstrip.js]: My bad. Now it should be right.

**Comment by kalkih** [comment]
Sorry, I've been on holiday for the last few days.

Looks great, merging now, thank you very much! 🎉

**Pull request merged**